authorization/role/service/role_management_service.go

```diff
@@ -46,6 +46,11 @@ func (s *roleManagementServiceImpl) ListByResource(ctx context.Context, currentI
 	return s.Repositories().IdentityRoleRepository().FindIdentityRolesByResource(ctx, resourceID, false)
 }
 
+// ListByResource lists all identity roles for the resource if the current user has permissions to view the roles
+func (s *roleManagementServiceImpl) ListByIdentityAndResource(ctx context.Context, currentIdentity uuid.UUID, resourceID string) ([]role.RoleDescriptor, error) {
+	return s.ListByIdentityAndResource(ctx, currentIdentity, resourceID)
```

Comment:
This method is
- broken (looks like an infinite recursive loop?)
- not covered by tests
- not used anywhere :)

Wonder if it was some temporal experiments you forgot to clean up.

Reply:
@xcoulon , have you taken care of this?

Reply:
oh.... my bad! Let me take care of it now!

Reply:
@sbose78 it's now been taken care of. See https://github.com/fabric8-services/fabric8-auth/pull/599/commits/3f93465f00366a503a8542d1fe357310213be32c